### ISSUE TITLE
status@190516-062034-d94424: download via aria2 failed

### ISSUE THREAD

**Issue by kaykouo** [ScoopInstaller/Extras#9272]
#### Scoop Configuration

```json
{
    "lastUpdate":  "2022-09-20T19:26:53.3389578+02:00",
    "SCOOP_REPO":  "https://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master"
}
```

ERROR Download failed! (Error 3) Resource was not found
ERROR https://status-im.ams3.cdn.digitaloceanspaces.com/StatusIm-190516-062034-d94424-release.exe#/dl.7z
    referer=https://status-im.ams3.cdn.digitaloceanspaces.com/StatusIm-190516-062034-d94424-release.exe#/
    dir=C:\Users\Admin\scoop\cache
    out=status#190516-062034-d94424#https_status-im.ams3.cdn.digitaloceanspaces.com_StatusIm-190516-062034-d94424-release.exe_dl.7z

ERROR & 'C:\Users\Admin\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\Admin\scoop\cache\status.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=2948 --continue --summary-interval=0 --auto-save-interval=1

**Comment by KrisSchneider** [comment]
Creating a fix for this manifest in #9309.

**Comment by KrisSchneider** [comment]
Sorry for the wait, this PR should fix this issue with the Status app.
